components/builder-api-client/src/artifactory.rs

```diff
@@ -447,6 +447,7 @@ impl BuilderAPIProvider for ArtifactoryClient {
                    pa: &mut PackageArchive,
                    token: &str,
                    _force_upload: bool,
+                   _disable_build_group: Option<&str>,
```

Comment:
I wouldn't optimize for the provider model as that is likely going away soon (since we;re not going the Artifactory plug in direction any longer)

Reply:
Good to know! Any reason not to take the Artifactory code out right away?

Reply:
The only reason to not take it out right away is to avoid complicating things in case we have to revert any of the other PRs that also touch that code (paranoia).